pythoncalculator/divide.py

```diff
@@ -0,0 +1,2 @@
+def divide(x, z):
+    return x / y

```

Comment:
```suggestion
    return x / z

```
Error noticed. Changed y to z